src/hooks/accounts/mutations/useSetActiveAccountMutation.tsx

```diff
@@ -1,21 +1,16 @@
 import { useMutation } from '@apollo/client';
 import { loader } from 'graphql.macro';
 import { GET_ACTIVE_ACCOUNT } from '../queries/useGetActiveAccountQuery';
-
-export const SET_ACTIVE_ACCOUNT = loader('./../graphql/SetActiveAccount.mutation.graphql');
+export const SET_ACTIVE_ACCOUNT = loader(
+  './../graphql/SetActiveAccount.mutation.graphql'
+);
 
 export interface SetActiveAccountMutationVariables {
-    id: string | undefined
+  id: string | undefined;
 }
 
-export const useSetActiveAccountMutation = (variables: SetActiveAccountMutationVariables) => useMutation<void, SetActiveAccountMutationVariables>(
-    SET_ACTIVE_ACCOUNT,
-    {
-        variables,
-        refetchQueries: [
-            // TODO: figure out why do we need to refetch active account manually,
-            // while all accounts query gets updated automatically
-            GET_ACTIVE_ACCOUNT
-        ]
-    }
-)
\ No newline at end of file
+export const useSetActiveAccountMutation = (options?: Object) =>
+  useMutation<void, SetActiveAccountMutationVariables>(SET_ACTIVE_ACCOUNT, {
+    ...options,
+    refetchQueries: [{ query: GET_ACTIVE_ACCOUNT }],
```

Comment:
new way to pass the list of queries after apollo update or did i miss something in the previous implementation?